modules/common/css/lib/errors.scss

```diff
@@ -1,149 +1,189 @@
 // Errors
-@mixin wdc-form-validation($_borderColor) {
-  border-color: $_borderColor;
+@mixin wdc-form-validation($_type) {
 
-  &.wdc-form-checkbox:not(.wdc-form-checked):not(:checked) + .wdc-form-label:before {
+  $_borderColor: #000 !default;
+
+  @if ($_type=='error') {
+    $_borderColor: $wdc-color-error;
+  }
+
+  @else if ($_type=='alert') {
+    $_borderColor: $wdc-color-alert-light;
+  }
+
+  &.wdc-form-textarea,
+  &.wdc-form-textinput,
+  &.wdc-form-select {
+    border-color: $_borderColor;
+    box-shadow: inset 0 0 0 1px $_borderColor;
+
+    @if ($_type=='alert') {
+      border-color: $wdc-color-alert-dark;
+      box-shadow: inset 0 0 0 2px $_borderColor;
+    }
+  }
+
+  &.wdc-form-checkbox:not(.wdc-form-checked):not(:checked)+.wdc-form-label:before {
     border-color: $_borderColor;
   }
 
   &.wdc-form-hover,
   &:hover {
+
     &.wdc-form-textarea,
     &.wdc-form-textinput,
     &.wdc-form-select {
       border-color: $_borderColor;
+
+      @if ($_type=='alert') {
+        border-color: $wdc-color-alert-dark;
+        box-shadow: inset 0 0 0 2px $_borderColor;
+      }
     }
   }
 
-  &.wdc-form-focus,
-  &:focus {
+  &.wdc-form-focus:not([disabled]):not(.wdc-form-disabled),
```

Comment:
in button css update, we removed the `wdc-...-disabled` class name and just used the native disabled attribute. Should we do the same for forms?

Reply:
@mannycarrera4 we didn't entirely remove the class from the scss, we just changed the stories to use the attribute instead of the class so they weren't focusable. Let's leave this as is.